src/corelibs/U2Formats/src/sqlite_dbi/SQLiteFeatureDbi.cpp

```diff
@@ -371,9 +364,6 @@ QSharedPointer<SQLiteQuery> SQLiteFeatureDbi::createFeatureQuery(const QString &
     if (useKeyTable) {
         tablesPart += ", FeatureKey AS fk";
     }
-    if (useRegion && (1 != fq.intersectRegion.length || oneClosestFeature)) {
-        tablesPart += ", FeatureLocationRTreeIndex AS fr";
```

Comment:
This line defines "fr" as a reference to the FeatureLocatoinRTreeIndex.
You removed this line but "fr" is still referenced inside of the query: see "order by" and other parts with "fr".

Unfortunately our tests do not catch this situation, so we need new tests that will 
- With the old code:  call the 'createFeatureQuery' the way that "fr" is present in the result query -> check that it works
- With the new code: call the 'createFeatureQuery' with the same params, check that the result code still works.

The best candidates here are API tests, where you can call the method directly.